README.md

```diff
@@ -101,6 +101,11 @@ $globalConfig->addAppInstance('local')
 return $globalConfig;
 ````
 
+> Hint:
+> If you ever need or want to rename the configuration file for phpsu, you can do this by providing 
+> an environment variable `PHPSU_CONFIG_FILE` and setting it to the file name including the file ending
+> example: `PHPSU_CONFIG_FILE=alternative-config.php`
```

Comment:
- [ ] remove?

Reply:
@Kanti I stumbled today on an example for where this is actually quite useful. We always were motivated to make the configuration as easy and readable as possible: I now have the following setup:
- two ssh connections (with proxy)
- one prod app instance and 6 dev app instances for TYPO3
   - including shared filesystem and shared database configuration
- two web-tools (3rd party managed php scripts, that we need locally)
   - same ssh connections but different filesystems (completly different paths remote)
   - this collides with the shared fileadmin
   - we only need this like every once in a while

the config is now pretty confusing, i dont want to have configuration commented out here, and having an env variable that controls this would be really silly (yet still a pretty straight forward way)
my idea would now be three files:
- phpsu-config.ssh.php (the ssh config that both configuration import from)
- phpsu-config.webtools.php (phpsu -c phpsu-config.webtools.php sync)
- phpsu-config.php (default)

for every normal aspect the user only imports TYPO3 but every blue moon through the different command i can import the other configuration

What do you think?